src/PayPal/Api/Subscription/BillingInfo/OutstandingBalance.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Subscription\BillingInfo;
+
+use Swag\PayPal\PayPal\Api\Plan\BillingCycle\PricingScheme\FixedPrice;
+
+class OutstandingBalance extends FixedPrice
```

Comment:
extend from https://github.com/shopwareLabs/SwagPayPal/blob/master/src/PayPal/Api/Common/Value.php

Reply:
not possible, see FixedPrice :D

Reply:
changed to Money